robottelo.properties.sample

```diff
@@ -25,6 +25,13 @@ ssh_key=
 # Admin password when accessing API and UI
 # admin_password=changeme
 
+# section for ssh client settings
+[ssh_client]
+# Time to wait for the ssh command to finish, in seconds
+# command_timeout = 300
```

Comment:
I'd better follow the same style as all the rest properties: no extra spaces, just
```
# command_timeout=300
```